cips/cip-24.md

```diff
@@ -0,0 +1,60 @@
+| cip | 24 |
```

Comment:
```suggestion
| cip | TBD |
```
hi @ninabarbakadze, can you please change filename to something like `cip-versioned-gas-schedule-variables.md` and then once it is approved for draft, we will assign it a number? thank you

Reply:
done!

Reply:
thank you! for context, this is just to avoid merge conflicts while this is in draft